packages/manager-components/package.json

```diff
@@ -28,7 +29,8 @@
     "babel-preset-react-app": "^10.0.1",
     "react": "18.2.0",
     "sass": "1.71.0",
-    "tailwindcss": "3.3.3"
+    "tailwindcss": "3.3.3",
+    "element-internals-polyfill": "^1.3.10"
```

Comment:
this one should be a devDependency

Reply:
Fixed, and moved to devDependencies in pci-users package.json too